src/Elcodi/Plugin/StoreSetupWizardBundle/Resources/config/services.yml

```diff
@@ -9,6 +9,7 @@ services:
             configuration_manager: @elcodi.manager.configuration
             product_repository: @elcodi.repository.product
             shipping_range_repository: @elcodi.repository.shipping_range
+            enabled_methods: %store.payment_bridge.enabled_methods%
```

Comment:
The way of solving this is adding this configuration as a plugin configuration element.